src/views/pages/customer/wishlist.twig

```diff
@@ -37,7 +37,14 @@
                       </div>
                   </div>
                   <div class="flex items-center rtl:space-x-reverse space-x-4">
-                      <salla-add-product-button roduct-status="{{ product.status }}" product-id="{{ product.id }}" product-type="{{ product.type }}" loader-position="center"  fill="outline" class="flex-grow sm:grow-0 w-full md:w-40">
+                      <salla-add-product-button 
+                        {{product.is_require_shipping ? 'required-shipping' : ''}}
+                        roduct-status="{{ product.status }}" 
+                        product-id="{{ product.id }}" 
+                        product-type="{{ product.type }}" 
+                        loader-position="center" 
+                        fill="outline" 
+                        class="flex-grow sm:grow-0 w-full md:w-40">
```

Comment:
No need here, it's required for the applePay, so it's required when you have: `quick-buy` attribute